### PR TITLE
Commenting out the special behavior for Ruby v2.1.1

### PR DESCRIPTION
**Pull request by dengelgit**
This removes the special behavior for ruby 2.1.1 only as default.

### CODE REVIEW

File: ruby/rbenv/vars/Debian.yml [hunk 1]
```diff
@@ -17,4 +17,4 @@ rbenv_package_names:
   - libgdbm-dev
 
 # only needed for ruby 2.1.1
-rbenv_ruby_configure_opts: 'RUBY_CONFIGURE_OPTS=--with-readline-dir="/lib/{{ ansible_architecture }}-linux-gnu/libreadline.so.6"'
+# rbenv_ruby_configure_opts: 'RUBY_CONFIGURE_OPTS=--with-readline-dir="/lib/{{ ansible_architecture }}-linux-gnu/libreadline.so.6"'
```
Comment (arBmind) [inline ruby/rbenv/vars/Debian.yml]: I do not like these kinds of comments.

**Review by arBmind — CHANGES_REQUESTED** [review]
Thank you for the PR!

There is the same comment in `RedHad.yml`.

One solution would be to add this as a default and rename the variable. So if someone needs this he can use the variable easily. He wont have to search for comments or know these options.

Please adopt this or something similar, to help our users and keep the code clean.

**Comment by dengelgit** [comment]
I moved this variable to defaults and added a ruby version check.

**Comment by mstarke** [comment]
Does the path work on Redhat as well as Debian?

**Comment by mstarke** [comment]
@arBmind please merge if the changes are ok with you.

**Pull request merged**